demoShared/diagnostic.config.js

```diff
@@ -0,0 +1,5 @@
+(function(angular){
+    angular.module('demo').config(function(DiagnosticSrvProvider){
+        DiagnosticSrvProvider.setDiagnosticExamIdGetter(14);
```

Comment:
in diagnostic WorkoutsDiagnosticFlowProvider.setDiagnosticSettings there's a configuration of diagnostic id already, maybe should require from there instead of duplicate config